test/end-to-end/vehicle.js

```diff
@@ -374,6 +374,69 @@ test('vehicle batch', async(t) => {
   t.is(location.meta.requestId.length, 36);
 });
 
+test('vehicle request - odometer', async(t) => {
+  const response = await t.context.volt.request(
+    'get',
+    'odometer',
+    {},
+    {
+      'sc-unit-system': 'imperial',
+    }
+  );
+  t.deepEqual(
+    _.xor(_.keys(response), [
+      'distance',
+      'meta',
+    ]),
+    [],
+  );
+
+  t.truthy(typeof response.distance === 'number');
+  t.truthy(response.meta.dataAge instanceof Date);
+  t.is(response.meta.requestId.length, 36);
+  t.is(response.meta.unitSystem, 'imperial');
+});
+
+test('vehicle request - lock', async(t) => {
+  const response = await t.context.volt.request('post', 'security', {
+    action: 'LOCK',
+  });
+
+  t.deepEqual(
+    _.xor(_.keys(response), [
+      'status',
+      'message',
+      'meta',
+    ]),
+    [],
```

Comment:
```suggestion
    _.keys(response), [
      'status',
      'message',
      'meta',
    ])
```
If you are just verifying that those keys exist, this is a sort of lengthy way to verify that. Here is a suggestion of a more readable to way to do the same thing

Reply:
Checking the response attributes was done this way in almost all of the other tests, and after investigating deepEquals and testing with dummy objects, I realized we use the xor because otherwise, deepEquals will fail if the properties are not in the same exact order. I'm not sure if the order of the properties are guaranteed to always be the same, but if they aren't, it would make sense to me to keep it. Thoughts?

Reply:
I just realized it would be the lodash method that orders the keys - the _.keys method doesn't guarantee any order ([](https://lodash.com/docs/#keys)) so I believe it would be necessary.

Reply:
okay cool! I though `deepEqual` sorted arrays but this makes sense :+1: